src/git-duet/git-duet-rebase/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"git-duet/internal/cmd"
+	"git-duet/internal/cmdrunner"
+)
+
+func main() {
+	err := cmdrunner.Execute(
+		cmd.New("rebase"))
```

Comment:
I don't think the ExecuteWithSignoff is needed, as no new commit is being made when you rebase.  It just keeps whatever author/committer metadata already existed on the commit(s) which were rebased.

I tried it out and it seems to work fine.